src/components/memorygame/gameHelpers.jsx

```diff
@@ -0,0 +1,34 @@
+// manipulates isFlipped and isMatched property of card a state
+// property example : "isMatched"
```

Comment:
you should change this comment to `cardStateKey example: ...`

Reply:
PTAL